src/Common/OpenTelemetryTraceContext.h

```diff
@@ -11,7 +11,12 @@ struct OpenTelemetryTraceContext
     // The incoming tracestate header and the trace flags, we just pass them
     // downstream. See https://www.w3.org/TR/trace-context/
     String tracestate;
+
+#if defined(OS_SUNOS)
+    uint8_t trace_flags = 0;
+#else
```

Comment:
This is probably by mistake, let's try to do this change for all platforms.

Reply:
Sounds good, I'll add a commit.